packages/ida.plugin.sigmaker.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    # Download the plugin
```

Comment:
[nitpick] I think this comment is not needed, as this package only download a single file and the code is IMO clear.